contracts/contracts/token/WOETH.sol

```diff
@@ -28,9 +29,28 @@ import { OETH } from "./OETH.sol";
 contract WOETH is ERC4626, Governable, Initializable {
     using SafeERC20 for IERC20;
     using StableMath for uint256;
+    using SafeCast for uint256;
+
     uint256 public oethCreditsHighres;
     bool private _oethCreditsInitialized;
-    uint256[48] private __gap;
+    uint128 public creditsPerTokenLimit;
+    uint128 public cptLimitEndTime;
+    uint256[47] private __gap;
+
+    uint256 public constant YIELD_INCREASE_CADENCE = 1 days - 1 hours;
+    /* This is the limit by which the yield of OETH is allowed to propagate to WOETH in a
+     * single YIELD_INCREASE_CADENCE. This prevents rapid increase of WOETH value by
+     * limiting a maximum instant jump in price. The 0.25% limit equates to 91% yearly
+     * maximum yield.
+     *
+     * If OETH has a daily yield that is larger than 91% the effective yield will spill over
+     * to WOETH in the next yield increase cadence.
+     *
+     * If the daily yield equating to 91% YOY yield isn't surpassed then the OETH rebases
+     * are in sync with WOETH. Resulting in no economical gain from wrapping OETH to WOETH
+     * and back to try to game the yield distribution.
+     */
+    uint256 public constant MAX_YIELD_INCREASE = 25e14; // 0.25%
```

Comment:
We are the blessed summer children who have never lived hyper-inflation.

But if the underlying currency is inflating faster than this limit allows, then our users will be in deep trouble. I think we just want to head off donations that damage lending platforms, rather than bet our user funds on stable economies.

So I'd run this limit something more like 3-5%.

Reply:
That is a very good point.